GOVERNANCE.md

```diff
@@ -0,0 +1,33 @@
+# Paraglider Governance
+This document outlines the governance of the project and will be added to soon. 
+
+The Technical Steering Committee (TSC) is in charge of making high-level technical decisions for Paraglider. 
+
+## Technical Steering Committee Members
+
+**Sarah McClure** </br>
+UC Berkeley
+
+**Diego Vega** </br>
+Microsoft
+
+**Sylvia Ratnasamy** </br>
+UC Berkeley
+
+**Deepak Bansal** </br>
+Microsoft
+
+**Pravein Govindan Kannan** </br>
+IBM Research
+
+**Ashok Narayanan** </br>
+Google
+
+**Anees Shaikh** </br>
+Google
+
+**Sean Kim** </br>
+Databricks
```

Comment:
```suggestion
UC Berkeley
```

I'm with Berkeley until 1st week of August, so I'll change once I start at Databricks.